### PR TITLE
move SCC to kube-apiserver

### PR DESCRIPTION
**Pull request by deads2k**
builds on https://github.com/openshift/origin/pull/23212

this removes the last origin dep from kube-apiserver: SCC

After this we can move the kas patches to our fork of kube.

/assign @sttts @damemi

### CODE REVIEW

File: pkg/security/apiserver/registry/podsecuritypolicysubjectreview/rest.go [hunk 1]
```diff
@@ -107,10 +108,11 @@ var scheme = runtime.NewScheme()
 
 func init() {
 	utilruntime.Must(securityapi.Install(scheme))
+	utilruntime.Must(securityv1.Install(scheme))
```
Comment (sttts) [inline pkg/security/apiserver/registry/podsecuritypolicysubjectreview/rest.go]: surprising to see in a registry. Why that?

Comment (deads2k) [reply pkg/security/apiserver/registry/podsecuritypolicysubjectreview/rest.go]: > surprising to see in a registry. Why that?

There's a helper function used below that identifies types for references.

File: pkg/cmd/openshift-kube-apiserver/admission/security/sccadmission/admission.go [hunk 2]
```diff
@@ -6,6 +6,8 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/openshift/origin/pkg/cmd/openshift-kube-apiserver/admission/security/securitycontextconstraints/sccmatching"
```
Comment (sttts) [inline pkg/cmd/openshift-kube-apiserver/admission/security/sccadmission/admission.go]: order

**Comment by sttts** [comment]
Looked briefly over it (getting late here). Some nits, looks good otherwise, but could have some more eyes on it from @damemi.

File: pkg/security/apiserver/registry/podsecuritypolicyreview/rest.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/openshift/origin/pkg/cmd/openshift-kube-apiserver/admission/security/securitycontextconstraints/sccmatching"
```
Comment (damemi) [inline pkg/security/apiserver/registry/podsecuritypolicyreview/rest.go]: order

**Comment by deads2k** [comment]
tagging based on comment.

**Pull request merged**